nef/AppDelegate.swift

```diff
@@ -48,10 +49,10 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         switch command {
         case .preferences:
             preferencesDidFinishLaunching()
-        case .carbon(let code):
-            carbonDidFinishLaunching(code: code)
-        case .clipboardCarbon(let code):
-            clipboardCarbonDidFinishLaunching(code: code)
+        case .exportSnippet(let selection):
```

Comment:
This should be `exportSnippetToFile`, as this name suggests this is a general case and the following one is a specific case from this one, but it's not, so the name is misleading.